about.css

```diff
@@ -0,0 +1,266 @@
+.background-aboutPage {
+  background: url(./assets/img/about-banner.png);
+  background-position: center center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-color: #f7f7f7;
+  width: 100%;
+  max-width: 1300px;
+  margin: auto;
+  height: 620px;
+  display: flex;
+}
+
+.intro-header-container-aboutPage {
+  text-align: center;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.image-title-aboutPage {
+  width: 60%;
+  margin-top: 90px;
+}
+
+.intro-text-aboutPage {
+  border: 1px solid lightgray;
+  width: 75%;
+  padding: 29px;
+  margin-top: 25px;
+  background-color: #fff;
+  margin-bottom: 35px;
+}
+
+.intro-text-aboutPage span {
+  color: gray;
+  font-size: 13px;
+  font-weight: 900;
+}
+
+.text-aboutPage {
+  font-weight: 900;
+  font-size: 14px;
+  color: #545454;
+  margin: 0 6px;
+}
+
+.emailto {
+  color: black;
+  font-weight: 900;
+}
+
+.background-MainProgram-aboutPage {
+  background-color: #fff;
+  width: 100%;
+  max-width: 1300px;
+  margin: auto;
+}
+
+.mainProgram-container-title-h1-aboutPage {
+  color: black;
+  font-size: 20px;
+  font-weight: 900;
+}
+
+.mainProgram-p-aboutpage {
+  text-align: center;
+  font-size: 12px;
+  font-weight: 600;
+}
+
+.winning-logo {
+  margin: 35px 0;
+}
+
+.winning-logo img {
+  border: 0.1px solid #ececec;
+}
+
+.border-top {
+  border-top: 1px solid lightgray;
+}
+
+.mainProgram-container-title-h1-summit {
+  color: black;
+  font-size: 20px;
+  font-weight: 900;
+  text-shadow: 0.1px 0 black;
+}
+
+.hr_submit_logo {
+  width: 7% !important;
```

Comment:
- [ ] Do not overuse !important rule in your CSS style definitions, please remove most or all the !important usage that you currently have to comply with the HTML and CSS best practices requirements.

Reply:
I have used "!important" in parts that were being affected by other container properties. I will remove some of them but in 850 lines of CSS, there are like 25 "!important"